docs/tech_primer.md

```diff
@@ -0,0 +1,157 @@
+# Smart-Ready Building Technical Primer
+
+This document provides  primer for smart-ready building assembly, to essentially
+answer the question *"What does Smart Ready mean?"* and *"How do we know it to be
+true?"* Specifically, it addresses the delta between a traditional site and
+smart-ready sites: places where new bleeding edge building technology is being
+used.
+
+# Key Terms & Definitions
+
+## Building Models
+
+There are different levels of "digital maturity" for different buildings. Each
+one has a slightly different scope in terms of what it means to be "complete."
+For the most part, this document addresses the scope of a Smart Ready building:
+-   **Legacy**: It works, but does not meet requirements or guidelines.
+-   **Compliant**: Meets basic security and networking requirements.
+-   **Smart Ready**: On-prem integrated with the cloud using UDMI & DBO.
+-   **Digital Building**: Completely integrated with back-end services.
+
+## Device
+
+Hardware that exists in the building that can change state in a way that can be machine communicated. A potentiometer or a flow switch is a device if it is monitored by a digital system. A commissioning valve or damper with no communication is not a device. 
+
+## Devices Classification
+
+Various adjectives qualify the different flavors of a "device" found in a building. More than one can apply, and each has a specific technical meaning/definition (i.e. they are not subjective).
+
+### Connectivity
+
+How data is moved around on-prem
+
+-   **Passive**: The device has only passive components (thermistor, dry contact)
+-   **Analog**: The device uses an analog connection (1-10V, 4-20mA etc)
+-   **Serial**: The device does not have an IP address, rather uses a wired serial connection.
+-   **Networked**: The device has its own IP address and presence on the wired or wireless IP network.
+-   **Bridge connected**: The device is on an isolated wired or wireless network behind a gateway device that is networked. 
+
+### Ingestion
+
+How data gets up into the cloud
+
+-   **Direct**: Maintains an authenticated connection directly to Cloud IoT Core.
+    -   Must be networked
+    -   Has a unique private auth key
+-   **Gateway**: A direct device that manages data for other proxied devices.
+    -   Bind & attach of proxied devices
+    -   Encapsulation anti-pattern
+-   **Proxied**: A device that is not direct and is managed through a gateway.
+    -   Proxied networked devices, e.g. BACnet/IP
+    -   Proxied non-networked devices, e.g. RS485, DALI
+-  **Cloud-Cloud**: A device is managed by a third party who provides a cloud connector
+    -   Approved third party managed service
+    -   UDMI virtual device server connects to third party cloud API
+
+### Representation
+
+How data is semantically represented
```

Comment:
mmm -- not quite sure "semantically" fits here, there still is no _meaning_ behind it. It's really something about how the data is incorporated into the data processing pipeline.

Reply:
"How devices are represented within different information sources"? 

Reading it again it's more about the device (logical, reporting, virtual) rather than data?

Reply:
I think it's about "device relationships and how they are modeled"... ?